Avoid adding four new packages that aren't necessary for the manual page,
and fix the path to the manual source to allow it to be generated.
--- a/doc/en/conf.py
+++ b/doc/en/conf.py
@@ -65,16 +65,12 @@
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = [
-    "pallets_sphinx_themes",
-    "pygments_pytest",
     "sphinx.ext.autodoc",
     "sphinx.ext.autosummary",
     "sphinx.ext.extlinks",
     "sphinx.ext.intersphinx",
     "sphinx.ext.todo",
     "sphinx.ext.viewcode",
-    "sphinx_removed_in",
-    "sphinxcontrib_trio",
 ]
 
 # Building PDF docs on readthedocs requires inkscape for svg to pdf
@@ -316,7 +320,7 @@
 
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
-man_pages = [("usage", "pytest", "pytest usage", ["holger krekel at merlinux eu"], 1)]
+man_pages = [("how-to/usage", "pytest", "pytest usage", ["holger krekel at merlinux eu"], 1)]
 
 
 # -- Options for Epub output ---------------------------------------------------
